launcher_scripts/conf/training/gpt3/1b_improved.yaml

```diff
@@ -124,6 +123,20 @@ model:
   fp8_amax_compute_algo: most_recent
   reduce_amax: true
   use_emha: false
+  gc_interval: 100 # Interval of the host memory garbage collection
+
+  ## Transformer Engine
+  transformer_engine: False # leaving this test without TE for fuctional test variety
```

Comment:
we can't currently use TE with the improved configs

Reply:
done